bin/GameIndex.dbf

```diff
@@ -1171,10 +1171,10 @@ Name   = Dead or Alive 2 - Hardcore
 Region = PAL-M5
 Compat = 1
 [patches = 7A51F86E]
-//Patched by Prafull
-//Some missing sounds
-patch=0,EE,002b4c44,word,24060000
-patch=0,EE,002b4dc0,word,10000014
+	comment=Patch by Prafull
+	// Some missing sounds.
```

Comment:
Are missing sounds a side effect of the patch or the issue being fixed. Needs some clarification.